internal/config/cardpick.go

```diff
@@ -0,0 +1,83 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/vinegarhq/vinegar/sysinfo"
+)
+
+// opt accepts the following values:
+// aliases - "integrated", "prime-discrete" and "none": Equivalent to "0", "1" or empty. Enables an extra "prime" check.
+// integer - GPU index
+// empty   - Skips logic and does nothing.
+func (b *Binary) pickCard() error {
+	opt := b.ForcedGpu
+
+	aliases := map[string]string{
+		"integrated":     "0",
+		"prime-discrete": "1",
+		"none":           "",
+	}
+
+	var aIdx string
+
+	prime := false
```

Comment:
Make this and `aIdx` part of one `var` clause. Booleans are already false by default.

Reply:
Fixed.